tools/packages.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-    <package id="Cake" version="0.28.0" />
+    <package id="Cake" version="1.1.0" />
```

Comment:
This is unrelated to the comment above, as it's about your build system - Make sense to update to the latest version if possible.

If you wanted to go one step further, you could move to the [Cake .NET Tool](https://cakebuild.net/docs/running-builds/runners/dotnet-tool) and use the [simplified bootstrappers](https://github.com/cake-build/resources/tree/master/dotnet-tool)